### PR TITLE
Adding meat to the lasagna

### LINKED ISSUE #16: Adding ingredients to the lasagna

### PR DESCRIPTION
**Pull request by remilaumont**
fixes #16

### CODE REVIEW

**Review by jakobtkn — CHANGES_REQUESTED** [review]
Nice change! However, in this case we should also change the name of the file to lasagna.md

**Comment by remilaumont** [comment]
Ah yes!  Maybe it is going to be to heavy in the stomach. What do you think @s194317  ?

**Comment by remilaumont** [comment]
Let us remove the meat.

**Comment by remilaumont** [comment]
I removed it now, could you have another look? I change the oven temperature and the coolling time.

**Review by jakobtkn — APPROVED** [review]
I think this is a good solution. Perhaps we can think about adding a meaty lasagna in the future. Thank you for your work @remilaumont !

File: mains/vegetarian-lasagna.md [hunk 1]
```diff
@@ -23,7 +23,7 @@
 ## Instructions
 
 1. **Preheat Oven:**
-   - Preheat your oven to 375°F (190°C).
+   - Preheat your oven to 375°F (60°C).
```
Comment (remilaumont) [inline mains/vegetarian-lasagna.md:26]:
```suggestion
   - Preheat your oven to 375°F (180°C).
```

File: mains/vegetarian-lasagna.md [hunk 2]
```diff
@@ -49,6 +49,6 @@
    - Remove the foil and bake for another 15-20 minutes, or until the cheese is bubbly and golden brown.
 
 6. **Serve:**
-   - Let the lasagna cool for 10-15 minutes before slicing.
+   - Let the lasagna cool for 17 minutes before slicing.
```
Comment (remilaumont) [inline mains/vegetarian-lasagna.md:52]:
```suggestion
   - Let the lasagna cool for 15 minutes before slicing.
```

**Pull request merged**